src/doc/src/reference/workspaces.md

```diff
@@ -6,36 +6,91 @@ various settings such as profiles. Packages that are part of a workspaces are
 called *workspace members*. There are two flavours of workspaces: as root
 package or as virtual manifest.
 
-### Root package
+The key points of workspaces are:
+
+* All packages share a common `Cargo.lock` file which resides in the
+  *workspace root*.
+* All packages share a common [output directory], which defaults to a
+  directory named `target` in the *workspace root*.
```

Comment:
Should the benefits of sharing a `Cargo.lock` and one `target` dir be explained?

Reply:
For output, let's rely on the information in that link.  For cargo.lock, let's mirror output and provide a link for more details.